src/includes/navigation-services.njk

```diff
@@ -4,11 +4,11 @@
         <nav class="navbar navbar-expand-lg  blur blur-rounded top-0  z-index-3 shadow position-absolute mt-4 my-3 py-2 start-0 end-0 mx-4">
           <div class="container-fluid">
             <a class="navbar-brand font-weight-bolder ms-sm-3" href="/" rel="tooltip" title="Designed and Coded by Creative Tim" data-placement="bottom">
-              Expansive Labs | {{ title }}
+      Expansive Labs
             </a>
-            <a target="_blank" 
+            {# <a target="_blank" 
```

Comment:
Suggest just removing buy now rather than commenting it out, I don't foresee us ever using a buy now button.